src/main/java/com/rcv/JsonParser.java

```diff
@@ -33,8 +33,9 @@
   // purpose: parse input json file into an object of the specified type
   // param: jsonFilePath path to json file to be parsed into java
   // param: valueType class of the object to be created from parsed json
+  // param: logsEnabled display log messages
   // returns: instance of the object parsed from json or null if there was a problem
-  static <T> T readFromFile(String jsonFilePath, Class<T> valueType) {
+  static <T> T readFromFile(String jsonFilePath, Class<T> valueType, boolean logsEnabled) {
```

Comment:
Boolean args tend to be confusing in public methods. Would prefer to expose a separate wrapper method `readFromFileWithoutLogging` for clarity.

Reply:
Changed, k picante!